packages/core/src/output-formats.ts

```diff
@@ -31,6 +31,38 @@ export const sourceFormat: OutputFormat = () => (metadatas) => {
   return metadatas.map((m) => metadataToSource(m))
 }
 
+export const imgFormat: OutputFormat = () => (metadatas) => {
+  let largestImage
+  let largestImageSize = 0
+  for (let i = 0; i < metadatas.length; i++) {
+    const m = metadatas[i]
+    if ((m.width as number) > largestImageSize) {
+      largestImage = m
+      largestImageSize = m.width as number
```

Comment:
I'm wondering (and whether that's actually a good idea) is if it's possible 
- to use the original image size as the one for `src/w/h`. Does that make sense? Since upscaling is lossy
- to cap the generation of images (and output meta) to "not more than 2 times the original size" because upscaling beyond that is probably wasteful (?)

Reply:
I was assuming that we would never upscale images. I just did a quick search and https://imagekit.io/blog/how-to-resize-image-in-html/ suggests the same

Reply:
> I was assuming that we would never upscale images.

Though I guess that needs to happen in `imagetools` because `@sveltejs/image` doesn't know the size of the original image. I'll check that and implement as a separate PR if it's not already happening